packages/build/src/tracing/main.ts

```diff
@@ -1,5 +1,5 @@
-import { HoneycombSDK } from '@honeycombio/opentelemetry-node'
```

Comment:
We don't really need the HoneycombSDK since we're using a collector. Plus it was logging warnings I couldn't supress:
- https://github.com/honeycombio/honeycomb-opentelemetry-node/blob/ed5b096b16bc88cc12b08d0c5e4a50c6df894140/src/honeycomb-options.ts#L109C1-L111